content/en/build/building-with-lotus-APIs.md

```diff
@@ -0,0 +1,646 @@
+   cd lotus/
+   git checkout <vX.X.X> # tag for a release, currently v1.15.0
+   ```
+
+2. Build and install Lotus for Calibnet
+
+   ```shell
+   make clean calibnet # Calibration with min 32GiB sectors
+   sudo make install
+   ```
+
+3. Fast sync using latest snapshot (still need several hours to fully sync the chain)
+
+   ```shell
+   lotus daemon --import-snapshot <path-to-calibration-snapshot>
+   ```
+
+4. [Create a wallet](https://lotu.sh/lotus/manage/manage-fil/#creating-a-wallet) and request some TestNet FIL from [Faucet](https://faucet.calibration.fildev.network/)
+
+4. [Enable remote API access to your Lotus node](https://lotu.sh/developers/api-access/)
+
+If everything set up correctly, you should be able to verify to:
+
+- make sure that everything is working correctly, check the status of your local node:
+
+  ```shell with-output
+  #Check if the node sync to the highest tipset
+  lotus sync wait
+  ```
+
+  ```shell
+  Worker: 10161; Base: 778820; Target: 778820 (diff: 0)
+  State: complete; Current Epoch: 778820; Todo: 0
+  
+  Done!
+  ```
+
+- Check your wallet balance
+
+  ```shell with-output
+  #Check wallet balance
+  lotus wallet list 
+  ```
+
+  ```shell
+  Address                                    Balance        Nonce  
+  t154xvuihhicgluafwmohwzwmtmqp44pwwgewyvma  100 FIL          0
+  ```
+
+### Create a node.js project
+
+Let's also set up a Node.js project with necessary dependencies, we will write basic javascript functions to focus more on learning the code to interact with Filecoin Network. 
+
+1. Create a new node.js project
+
+   ```shell
+   mkdir build-with-lotus && cd build-with-lotus
+   npm init #Initialize an project
+   ```
+   
+2. Install the fielcoin.js library
+
+   ```shell
+   npm install filecoin.js
+   ```
+
+
+### Test a simple function call
+
+Now you're ready to write code to connect to your Lotus node and interact with the Lotus APIs. Let's write a simple script to query the Lotus version to verify if everything works as expected.
+
+1. Create a `chainDataQuery.mjs` file in your project and add the following code. We use the `mjs` file format because it supports import statements.
+
+    ```javascript
+    //1. Import necessary packages
+    import { HttpJsonRpcConnector, LotusClient } from "filecoin.js";
+
+    //2. Use the local node URL to create a connection to your Lotus node
+    const localNodeUrl = "http://127.0.0.1:1234/rpc/v0";
+    const localConnector = new HttpJsonRpcConnector({ url: localNodeUrl });
+
+    //3. lotusClient exposes all Lotus APIs
+    const lotusClient = new LotusClient(localConnector);
+    const version = await lotusClient.common.version();
+    console.log(version);
+    ```
+
+2. Run the following command in your project directory:
+
+    ```shell with-output
+    ~/build-with-lotus $ node basicQuery.mjs
+    ```
+
+    ```shell
+    {
+      Version: '1.15.0+calibnet+git.0ac1bbc7a',
+      APIVersion: 66816,
+      BlockDelay: 30
+    }
+    ```
```

Comment:
The `with-output` class isn't enabled on this site. Instead, write something like:

````
```shell
cat text.txt
```

This should output something like:

```shelll
Hello world!
```
````

Reply:
This will add a lot more unnecessary text in the tutorial. It is quite obvious that those blocks are output so that I will leave it as it is now.